trader/trader.go

```diff
@@ -180,6 +191,10 @@ func (t *Trader) deleteAllOffers(isAsync bool) {
 			log.Fatalf("%scontinuing to exit after showing error during submission of delete offer ops: %s", logPrefix, e)
 			return
 		}
+
+		t.threadTracker.TriggerGoroutine(func(inputs []interface{}) {
+			t.metricsTracker.SendDeleteEvent(false)
```

Comment:
we are eating up the error from the return value here, we should check the error and log like we do for the `SendUpdateEvent() call on line 125`